dev-lang/mujs/mujs-1.0.5.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```

Comment:
IIRC, idiomatic EAPI declaration is unquoted.

Reply:
Ok. I've seen both ways.